back/mailer.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+//step1
+const nodemailer = require('nodemailer');
+const sendEmail = function(viesti,sposti) {
+  try{    
+    // declare vars,
+    //editing the reciever to suit sending the email
+    let cutEmail = sposti.slice(11, -3);
+    let fromMail = process.env.EMAIL;
+    let toMail = cutEmail;
+    let subject = 'Tasera info';
+    let text = viesti;
+
+    switch (viesti) {
+    case 'assigned':
+      text = 'Hei teille on annettu valvoja vuoro. Voitte nyt käydä vahvistamassa vuoronne.';
```

Comment:
Suggestion:

"Hei,

teille on annettu valvojavuoro. Voitte nyt käydä vahvistamassa vuoronne.

Terveisin TASERA ry."

It always feels bad for correcting spelling, I'm sorry for that!

Reply:
done